clients/java/signalr/src/main/java/InvocationMessage.java

```diff
@@ -1,8 +1,8 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-public class InvocationMessage {
-    int type = 1;
+public class InvocationMessage extends HubMessage {
+    int type = HubMessageType.INVOCATION.value;
```

Comment:
JsonHubProtocol.writeMessage will have to be rewritten eventually and when that happens you can change this to HubMessageType then